src/OpenTelemetry.Contrib.Extensions.AWSXRay/README.md

```diff
@@ -68,8 +68,6 @@ service namespace, instance id, service version.
 - **AWSECSResourceDetector**: cloud provider, cloud platform, container id.
 - **AWSEKSResourceDetector**: cloud provider, cloud platform, cluster name,
```

Comment:
Are these detectors not removed from this package?

Reply:
Fixed a5e35d30627f3757b622aa9159ad22b9f0d3fbff